spec/classes/init_spec.rb

```diff
@@ -3,10 +3,6 @@
   mandatory_facts = {
     :fqdn                => 'no-hiera-data.example.local',
     :test                => 'no-hiera-data',
-    :environment         => 'rp_env',
```

Comment:
we dont need this magic environment anymore? great!

Reply:
it moved to ./spec/spec_helper.rb as it is needed by `yum::repo`. But it haven't to be 'rp_env' anymore :)